openreview/conference/templates/tabsConferenceWebfield.js

```diff
@@ -153,7 +153,7 @@ function renderContent(notes, userGroups, activityNotes, authorNotes) {
       $('#your-consoles .submissions-list').append([
         '<li class="note invitation-link">',
           '<a href="/group?id=' + AREA_CHAIRS_ID + '" >',
-          AREA_CHAIRS_NAME.replace('_', ' ') + ' Console',
+          AREA_CHAIRS_NAME.replace(/_/g, ' ') + ' Console',
```

Comment:
It seems like in some of the webfields you are using the variable `AREA_CHAIRS_NAME` and other you're using `AREA_CHAIR_NAME`. I'd rather just use 1 variable, or if they need to be different make the names more different.

Reply:
AREA_CHAIR_NAME is the name of the group id, like: Senior_Programm_Committee. I think I have to use the prettyId here.